src/resources/dist/js/login.js

```diff
@@ -103,7 +103,7 @@ var checkInputLabels = function () {
         }
     };
 
-    $loginInput.on('keyup', function() {
+    $loginInput.on('keyup paste propertychange change click input paste', function() {
```

Comment:
not sure if propertychange and input are valid events, see https://www.quirksmode.org/dom/events/

Reply:
`input` is valid from FF only ( but that is where i was able to reproduce the error)
`propertychange` is normally used for radio or checkboxes, i could remove is due the paste is important